### DISCUSSION TITLE
Website Issues

### DISCUSSION THREAD

**Discussion by EAwesomeK** [eclipse-birt/birt#1206] [Q&A]
Are there known issues with the web site links?  For example, this URL  

https://www.eclipse.org/birt/documentation/integrating/viewer-usage.php

Drops you here:

https://eclipse.github.io/birt-website/

There are many occurrences of this type of broken link.

**Comment by wimjongman** [comment]
Thanks for reporting. Where did you get this link?

**Reply by wimjongman** [reply]
This is the correct link: https://eclipse.github.io/birt-website/docs/integrating/viewer-usage/

**Reply by hvbtup** [reply]
There are many old links in internet which point directly to specific (now outdated) pages ("deep links").
AFAIK these old deep links are now redirected to the landing page of the new documentation.
I don't think it's possible to do anything better than that.

**Reply by wimjongman** [reply]
Well, if there are links in the documentation that we provide (e.g. eclipse help) then we might have to look into that.

**Reply by hvbtup** [reply]
That would require to change the eclipse main website in such a way that for each formerly-existing web page a deep-link redirect to the most corresponding new web page is defined. I don't think this is possible with reasonable effort.
BTW Please note that I reported this same issue quite a long time ago shortly after the new BIRT web page was introduced.
After that, the redirect to the new web site was set up.

**Reply by wimjongman** [reply]
I mean in the F1 help of Eclipse.
